### PR TITLE
CIV-10909 request for reconsideration notification

### PR DESCRIPTION
**Pull request by Omaira-Melo-Hmcts**
### JIRA link (if applicable) ###
[CIV-10909 New property for send a notification in the reconsideration process, when the claim is updated](https://tools.hmcts.net/jira/browse/CIV-10909)

### Change description ###

An email notification is to be sent when the judge rules that the original SDO created and reviewed under the 'request for reconsideration' process has been upheld.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg.json
Comment (ShwetaTandel-hmcts) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg.json:1]: Should these changes be in non-prod files?

File: ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2.json
Comment (ShwetaTandel-hmcts) [inline ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2.json:1]: Should these changes be in non-prod files?

**Review by bhagyashreesharma90 — APPROVED** [review]

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg.json
Comment (Omaira-Melo-Hmcts) [reply ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg.json:1]: You are correct. I have refactored the files

File: ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2.json
Comment (Omaira-Melo-Hmcts) [reply ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2.json:1]: You are correct. I have refactored the files. Thank you

File: ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg-nonprod.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
+[
+  {
```
Comment (Azam-Hmcts) [inline ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEventCaseProg-nonprod.json:2]: There is already existing file "AuthorisationCaseEvent-SDO-R2-nonprod.json" move your changes to that file. FYI: Our all changes should be in nonprod files and its name should contain "SDO-R2".

File: ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2-nonprod.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
+[
+  {
```
Comment (Azam-Hmcts) [inline ccd-definition/CaseEvent/Camunda/NotificationEventsSDO-R2-nonprod.json:2]: Please change file name to "NotificationEvents-SDO-R2-nonprod.json" so its quickly identifiable when we do release of this feature.

**Review by bhagyashreesharma90 — APPROVED** [review]

**Review by Azam-Hmcts — APPROVED** [review]

**Pull request merged**